### PR TITLE
Update single priority no timeslice to sync with JAMA

### PR DESCRIPTION
**Pull request by chinglee-iot**
Description
-----------

Test Steps
-----------

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c [hunk 1]
```diff
@@ -450,8 +450,10 @@ void test_priority_change_tasks_different_priority_lower( void )
     vTaskGetInfo( xTaskHandles[0], &xTaskDetails, pdTRUE, eInvalid );
     TEST_ASSERT_EQUAL( 1, xTaskDetails.xHandle->uxPriority );
 
-    /* Verify the task remains running. When priority dropped in prvSelectHighestPriorityTask.
-     * All the idle core will yield for context switch. The ready queue is a FIFO. Task 0 will be choosed
+    /* Verify the task remains running. */
+    /* When priority dropped in prvSelectHighestPriorityTask, all the idle cores
+     * will yield for context switch. The ready queue is a FIFO. Core 0 will choose
+     * The first task in the ready queue which is task 1. Task 0 will be choosed
```
Comment (joshzarr) [inline FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c]: Typo, should be 'chosen' or 'selected'

File: FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c [hunk 2]
```diff
@@ -617,7 +619,7 @@ void test_task_create_tasks_lower_priority( void )
  * 
  * Task N - 1        New Task
  * Priority – 1      Priority – 2
- * State - Ready	 State - Running (Core N)
+ * State - Ready	 State - Running (First available core)
```
Comment (joshzarr) [inline FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c:622]: JAMA should be updated to match this change

**Review by joshzarr — APPROVED** [review]
Looks good! Approved with a couple minor comments

File: FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c (hunk 1 above)
Comment (chinglee-iot) [reply FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c]: Thanks for the comment. Update in new commit.

File: FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c (hunk 2 above)
Comment (chinglee-iot) [reply FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c:622]: Update in JAMA.

**Pull request merged**